examples/workflows/upstream/suspend-template.upstream.yaml

```diff
@@ -32,7 +32,7 @@ spec:
 
   - name: delay
     suspend:
-      duration: "20"    # Must be a string. Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
```

Comment:
Is this change intentional?

Reply:
It's synced directly from upstream so we don't control it.